network/edge-services/how-to/subscribe-edge-services.mdx

```diff
@@ -0,0 +1,71 @@
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+
+<Message type="tip">
+You can subscribe to Edge Services from the Object Storage or Load Balancer dashboards of the Scaleway console. However, the instructions below detail how to describe from the dedicated Edge Services section of the console. Whichever path you use, the same pricing plans and subscription mechanisms apply. Subscribing from the Object Storage or Load Balancer dashboards does **not** limit you to only creating pipelines for that product with your subscription.
+</Message>
+
+1. Click **Edge Services** in the **Network** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+
+2. Click **Subscribe to Edge Services**.
+
+    Information about available [pricing plans](https://www.scaleway.com/en/pricing/network/#edge-services) is displayed. All pricing plans have a fixed monthly price, and include a certain number of pipelines and a certain amount of cache data (data egressed from all your Edge Services pipelines' caches). Any extra pipelines or cache egress data are charged at the additional rates shown.
+
+3. Select a plan, and click **Subscribe to Edge Services**.
+
+    Your subscription is created, and you are returned to the Edge Services dashboard, on the **Pipelines** tab. All your Edge Services pipelines will display here, along with an indicator of your pipeline and cache consumption in relation to your monthly plan limits.
+
+    <Lightbox src="scaleway-edge-pipelines.webp" alt="The Edge Services dashboard pf the Scaleway console, where the Pipeline tab is in view. At the top are 'Pipelines used' and 'Cache used' indicator bars, which show how much of your plan's limits you have consumed so far this month." />
+
+## How to change your subscription plan
+
+You can change your Edge Services subscription plan at any time. Subscriptions are pro-rata: the monthly price will be applied based on the proportion of the month you are subscribed to the plan. 
+
+If you upgrade your plan, any additional charges (for extra pipelines or cache data) that you had already accumulated before the time of upgrade will still stand.
+
+Find out more about how Edge Service subscription plans and billing works, including scenarios for changing your plan mid-month, on our [Understanding Edge Services pricing](/network/edge-services/reference-content/understanding-edge-pricing/) page.
+
+1. Click **Edge Services** in the **Network** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your Edge Services dashboard displays.
+
+2. Click the **Plans** tab. An overview of your current plan and consumption displays, along with options to manage your plan or terminate your subscription.
```

Comment:
We won't display the current consumption at first (but it will be available quickly after the GA)